changelog/fragments/1763040576-fix-possible-data-corruption-in-tcp-and-syslog-inputs.yaml

```diff
@@ -9,11 +9,11 @@
 # - security: impacts on the security of a product or a user’s deployment.
 # - upgrade: important information for someone upgrading from a prior version
 # - other: does not fit into any of the other categories
-kind: feature
+kind: bug-fix
 
 # REQUIRED for all kinds
 # Change summary; a 80ish characters long description of the change.
-summary: Improve logging of cache processor and add ignore failure option.
+summary: Fix possible data corruption in tcp and syslog inputs
```

Comment:
I think we should add unix to the list of possibly impacted inputs.

```
filebeat/inputsource/unix/server.go
61:             factory := streaming.SplitHandlerFactory(inputsource.FamilyUnix, log, MetadataCallback, nf, splitFunc)
```

Reply:
Absolutely right, thank you! Somehow I missed this input when checking the code.